src/lib/ui/thermostat/ClimatePresets.tsx

```diff
@@ -40,7 +41,19 @@ export function ClimatePresets(props: ClimatePresetsManagement): JSX.Element {
     climatePresetKeySelectedForDeletion,
     setClimatePresetKeySelectedForDeletion,
   ] = useState<ThermostatClimatePreset['climate_preset_key'] | null>(null)
-  const deleteMutation = useDeleteThermostatClimatePreset()
+
+  const { mutate, isError, error, isPending } =
+    useDeleteThermostatClimatePreset()
+
+  const errorMessage = useMemo(() => {
```

Comment:
Why does this need `useMemo`?

Reply:
Yeah you're right this is not an expensive operation.